src/Service/MailService.php

```diff
@@ -94,15 +95,38 @@ public function send(): ResultInterface
         }
 
         //save copy of sent message to folders
-        if ($this->mailOptions->getSaveSentMessageFolder()) {
+        if ($this->mailOptions->getTransport() == 'Laminas\Mail\Transport\Smtp' &&
```

Comment:
Although it works just fine the way it's done here, but for consistency it would be better to check against `Laminas\Mail\Transport\Smtp::class` (just like in the config file)

Reply:
i checked the exact value that ends up when $this->mailOptions->getTransport() is called
it doesn't contain the ':class' part

Reply:
ok, i'll use Laminas\Mail\Transport\Smtp::class instead